aemrules.properties

```diff
@@ -1,14 +1,20 @@
 category=External Analysers
 description=Adds rules for AEM Java development
 homepageUrl=https://github.com/Cognifide/AEM-Rules-for-SonarQube
-archivedVersions=0.8,0.9,0.10,0.11
-publicVersions=1.0
+archivedVersions=0.8,0.9,0.10,0.11,1.0
+publicVersions=1.1
 
 defaults.mavenGroupId=com.cognifide.aemrules
 defaults.mavenArtifactId=sonar-aemrules-plugin
 
+1.1.description=Java Plugin 6.2 support
+1.1.sqVersions=[7.9,LATEST]
+1.1.date=2020-04-02
+1.1.changelogUrl=https://github.com/Cognifide/AEM-Rules-for-SonarQube/tree/v1.1
```

Comment:
This doesn't go to a change log. Did you mean this instead?
https://github.com/Cognifide/AEM-Rules-for-SonarQube/releases/tag/v1.1

Reply:
Thanks! Fixed.